src/tools/miri/src/shims/native_lib/trace/parent.rs

```diff
@@ -500,7 +500,8 @@ fn handle_segfault(
     capstone_disassemble(&instr, addr, cs, acc_events).expect("Failed to disassemble instruction");
 
     // Move the instr ptr into the deprotection code.
-    #[expect(clippy::as_conversions)]
+    #[allow(unknown_lints)]
```

Comment:
When you add things that are needed only for bootstraping, please mark them with `cfg_attr(bootstrap)` so that they don't unnecessarily stick around longer than they have to.

Reply:
Wasn't sure it was used for tools outside `compiler` folder. Noted for next time.